javascript/apps/taiga/src/app/modules/project/feature-kanban/components/create-status/kanban-create-status.component.html

```diff
@@ -0,0 +1,83 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<ng-container *transloco="let t">
+  <div
+    class="create-status"
+    #createStatusWrapper
+    *hasPermission="['create']; entity: 'story'">
+    <ng-container *ngIf="showAddForm; else addStatusTpl">
+      <form
+        class="form-card"
+        (tuiActiveZoneChange)="leaveForm($event)"
+        [showFormErrors]="submitted"
```

Comment:
Maybe you can do this? (I'm not 100% sure if it's works in your use case)

```html
      <form
       #form="ngForm"
        class="form-card"
        (tuiActiveZoneChange)="leaveForm($event)"
        [showFormErrors]="form.submitted"
```

Reply:
It wasn't working